02-set-up.Rmd

```diff
@@ -115,12 +119,10 @@ process. This can be started in modern Windows versions by typing `Ctrl-Alt-Del`
 #### Exercises {-}
 
 1. What is the exact version of your computer's operating system?
-2. Start an activity monitor then execute the following code chunk.
-In it `lapply()` (or its parallel version `mclapply()`) is used to *apply* a function, `median()`, over every column in the data frame object
-`X` (see Section \@ref(the-apply-family) for more on the 'apply family' of 
-functions).[The
-reason this works is that a data frame is really a list of vectors, each vector forming a column.
-]
+2. Start an activity monitor then execute the following code chunk in it.
+`lapply()` (or its parallel version `mclapply()`) is used to *apply* a function, `median()`, over every column in the data frame object
+`X` (see Section \@ref(the-apply-family) for more on the 'apply family' of functions) (The reason this works is that a data frame is really a list of vectors, each vector forming a column).
+
```

Comment:
No problem with shifting the footnote to the main body with brackets, but the punctuation in your suggestion is not right.